### ISSUE TITLE
[FEAT]: Detect ServerApps when Cosmos is installed on existing environment

### ISSUE THREAD

**Issue by Peiler10** [azukaar/Cosmos-Server#226]
### Feature Description

As a user I would like to see Cosmos detecting my existing media server apps (e.g Sonarr, Radarr etc) and in marketplace to see what I already have as installed

**Comment by azukaar** [comment]
Cosmos will show you your existing apps already, or do you mean showing a badge in the marketplace itself?

**Comment by BearTS** [comment]
Dormant Issue, should this be closed? @azukaar

**Comment by azukaar** [comment]
Yeah I mean we could close it, the only reason I didn't is because we could also implement a badge in the market that shows an app has been installed already, although it's not a super useful feature in my opinion to be honest, the ticket can be use to further discuss it
